### ISSUE TITLE
Support EDAM terms as an alternative to URNs for indicating model languages

### ISSUE THREAD

**Issue by jonrkarr** [SED-ML/sed-ml#94]
SED-ML relies on URNs to indicate the language of each model. This functions much like a dictionary which defines a set of recognized languages. 

At present, the curated dictionary is limited. It encompasses few languages, and it only captures an id (plus version) of the indicated language. This dictionary is also separate from larger community efforts to catalogue formats such as EDAM and FAIRSharing. Its also separate from specification URLs used by COMBINE.

I suggest that Model/@language be extended to allow the use of EDAM ontology terms to indicate model formats. EDAM is much larger. It already includes terms or has terms in the proposal pipeline for all of the existing SED language URNs. EDAM is actively maintained. New terms can be suggested through GitHub issues. Currently, EDAM is separate from FAIRSharing, but there are plans to mirror their content. In addition, EDAM can capture much more information about each language such as links to documentation, examples, licences, citations, etc.; relationships between languages and their parent formats; and the type of model that each format can represent. This includes tracking which languages are serialized to XML. This would bring SED-ML in line with a broader community effort.

The use of EDAM terms for model languages would be similar to the use of KiSAO terms for algorithms and their parameters.

For backwards combability, URNs could still be used. To help with compatibility, an URN to EDAM id map could be added to libSED-ML or served from a particular URL.

**Comment by matthiaskoenig** [comment]
Sounds good to me. 
We could restrict to a subset of terms, such as 
https://www.ebi.ac.uk/ols/ontologies/edam/terms?iri=http%3A%2F%2Fedamontology.org%2Fformat_2013&viewMode=All&siblings=false

**Comment by jonrkarr** [comment]
Yes, a particular parent node or relationships to one node could be used to direct SED-ML users to "recognized" languages. It might make sense to suggest creating a "topic" term for SED-ML, and suggest adding relationships from the "recognized" languages to this topic term. The topics section of EDAM is separate from the inheritance tree of formats. Using a topic term would avoid needing to suggest that model languages inherit from SED-ML.

**Comment by luciansmith** [comment]
Is this an L1v4 or an L2 suggestion?

**Comment by jonrkarr** [comment]
This can be left to L2.
